src/main/resources/bootstrap.yaml

```diff
@@ -18,4 +18,7 @@ spring:
         sscs-asb-primary-send-and-listen-shared-access-key: AMQP_PASSWORD
         docmosis-endpoint: PDF_SERVICE_BASE_URL
         docmosis-api-key: PDF_SERVICE_ACCESS_KEY
+        robotics-email-to: ROBOTICS_EMAIL_TO
```

Comment:
Is this needed in this file?

Reply:
Yes. Probably we don't have functional tests that send emails. When the team decides to use helm charts to deploy to AAT, it would be needed.

Reply:
ok